### PR TITLE
Fix html signature format balloon

### PR DESCRIPTION
**Pull request by GretaD**
Signed-off-by: GretaD <gretadoci@gmail.com>

### CODE REVIEW

**Comment by ChristophWurst** [comment]
on a side note I think that means we have to be double-careful with future updates of CKEditor as the css selectors could change or their base styles that we have to enforce here

File: src/components/AccountSettings.vue [hunk 1]
```diff
@@ -37,7 +37,7 @@
 		</AppSettingsSection>
 		<AppSettingsSection :title="t('mail', 'Signature')">
 			<p class="settings-hint">
-				{{ t('mail', 'A signature is added to the text of new messages and replies.') }}
+				{{ t('mail', 'A signature is added to the text of new messages and replies. Select the text to format it.') }}
```
Comment (GretaD) [inline src/components/AccountSettings.vue:40]: I think this line makes it more friendly for people to understand how to format it

**Review by ChristophWurst — APPROVED** [review]

**Comment by webcoreinc** [comment]
Where is the email sig setting in NC? I can't find this anywhere.

**Comment by ChristophWurst** [comment]
@webcoreinc https://help.nextcloud.com/c/apps/mail/35 is the place for questions.

**Pull request merged**